src/System.Data.OleDb/src/OleDbDataReader.cs

```diff
@@ -1166,7 +1166,7 @@ public override int GetValues(object[] values)
             {
                 throw ADP.ArgumentNull("values");
             }
-            MetaData info = DoValueCheck(0);
+            _ = DoValueCheck(0);
```

Comment:
@stephentoub  Style thoughts? I think generally we do not assign to a "discard" unless the compiler requires it (ie., for a property getter.. although one would hope we would fix the getter)

Reply:
We shouldn't need the discard.  We only use it in special situations, e.g. where we're working around a compiler/analyzer warning, or where we're otherwise in a situation where knowing that the result being ignored is important.  This isn't such a case.